.storybook/decorators/withHits.ts

```diff
@@ -19,7 +19,6 @@ export const withHits = (
     appId = 'latency',
     apiKey = '6be0576ff61c053d5f9a3225e2a90f76',
     indexName = 'instant_search',
-    searchParameters = {},
```

Comment:
Any additional parameters can be provided to a `configure` widget at the story level.

Reply:
we could change this to use a configure widget automatically if the parameter is given, if it's sufficient amount of work?

Reply:
It already uses a `configure` widget for the static parameters. If you want to add more values you have to create another `configure` widget. I don't see the value of adding an extra parameter only for this.